backend/core/src/listings/db/listing-query-builder.ts

```diff
@@ -50,7 +50,16 @@ export class ListingsQueryBuilder extends SelectQueryBuilder<Listing> {
     }
 
     for (const orderByCondition of orderByConditionDataArray) {
-      this.addOrderBy(orderByCondition.orderBy, orderByCondition.orderDir, orderByCondition.nulls)
+      if (orderByCondition.orderBy === "listings.status") {
+        const orderStr =
+          orderByCondition.orderDir === "ASC"
+            ? `CASE WHEN ${orderByCondition.orderBy} = 'pending' THEN 1 WHEN ${orderByCondition.orderBy} = 'active' THEN 2 WHEN ${orderByCondition.orderBy} = 'closed' THEN 3 END`
```

Comment:
Are we able to change the statuses to use the enum? so like `${ListingStatus.pending}` instead of `'pending'`